config/_default/menus/main.en.yaml

```
@@ -5699,6 +5699,11 @@ menu:
       parent: automation_pipelines
       identifier: automation_pipelines_inbox
       weight: 10002
+    - name: Set a Due Date
```

Ignore me if this doesn't match the topic title on purpose!
```suggestion
    - name: Set Due Date Rules
```

---

Whoops, left single comment accidentally; review forthcoming